### ISSUE TITLE
Incorrect installation - bad formatting of error message

### ISSUE THREAD

**Issue by DrRaphello** [MuMech/MechJeb2#957]
I had messed up my mechjeb installation and this message at least informed me about it.

This error message should properly display the problematic path (it's hidden now behind OK button) 

![obraz](https://user-images.githubusercontent.com/25507737/33124820-9d9ded04-cf7e-11e7-88c8-fa38ea54f751.png)

Needless to say, I've already fixed the installation issue

**Comment by sarbian** [comment]
It displays fine here. It must do that when using some specific display settings. Did you change the UI scale or use some unusual resolution ?

**Comment by DrRaphello** [comment]
I'm using 1920x1200 resolution with UI set to default 100%. I've just reproduced this by copying MJ2 whole folder to a folder of another mod (exactly what I did before by accident). I chose a folder with a long name - "C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\GameData\NavBallDockingAlignmentIndicatorCE".

My guess would be there's formatting issue with a long path name like this.
